### PR TITLE
Incorporated changes from dW testing table

### PR DESCRIPTION
**Pull request by sfdomina**

### CODE REVIEW

File: Lab 0/README.md [hunk 1]
```diff
@@ -42,25 +42,22 @@ To view a local version of the Kubernetes dashboard and to deploy apps into your
...
 
 1. Move the executable file to the /usr/local/bin directory using the command `mv /<path_to_file>/kubectl /usr/local/bin/kubectl` .
 
 2. Make sure that /usr/local/bin is listed in your PATH system variable.
 
- ```txt
- echo $PATH
- /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
- ```
+ ```echo $PATH          /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin```
```
Comment (MHBauer) [inline Lab 0/README.md]: No, this is a command being run followed by output.

Should switch the verbatim block from txt to console and then add a $ before the command.

Comment (jkomg) [reply Lab 0/README.md]: @sfdomina can you make that change and re-request?

Comment (sfdomina) [reply Lab 0/README.md]: yes, corrected the change and re-requested pull. thanks, @MHBauer and @kennjason.

**Review by jkomg — APPROVED** [review]
lgtm

**Pull request merged**